test/integration-legacy/teacher-registration/teacher_registration_utils.js

```diff
@@ -9,89 +9,89 @@ module.exports.constants = {
-    var consentCheckbox = driver.findElement(seleniumWebdriver.By.name('phoneConsent'));
-    var nextStepButton = driver.findElement(seleniumWebdriver.By.xpath(module.exports.constants.nextStepXpath));
-    var phoneNumberPromise = phoneInput.sendKeys('6172535960');
-    var consentPromise = consentCheckbox.click();
-    return Promise.all([phoneNumberPromise, consentPromise]).then(function () { // eslint-disable-line no-undef
-        nextStepButton.click().then(function () {
+    const phoneInput = driver.findElement(seleniumWebdriver.By.xpath('//input[@type="tel"]'));
+    const consentCheckbox = driver.findElement(seleniumWebdriver.By.name('phoneConsent'));
+    const nextStepButton = driver.findElement(seleniumWebdriver.By.xpath(module.exports.constants.nextStepXpath));
+    const phoneNumberPromise = phoneInput.sendKeys('6172535960');
+    const consentPromise = consentCheckbox.click();
+    return Promise.all([phoneNumberPromise, consentPromise]).then(() => { // eslint-disable-line no-undef
+        nextStepButton.click().then(() => {
             driver.wait(seleniumWebdriver.until
                 .elementLocated(seleniumWebdriver.By.className('organization-step')));
         });
     });
 };
 
 module.exports.fillOrganizationSlide = function (driver, seleniumWebdriver) {
-    var organizationInput = driver.findElement(seleniumWebdriver.By.name('organization.name'));
-    var titleInput = driver.findElement(seleniumWebdriver.By.name('organization.title'));
-    var typeCheckbox = driver.findElement(seleniumWebdriver.By.xpath('//input[@type="checkbox" and @value="3"]'));
-    var nextStepButton = driver.findElement(seleniumWebdriver.By.xpath(module.exports.constants.nextStepXpath));
-    var organizationPromise = organizationInput.sendKeys('MIT Media Lab');
-    var titlePromise = titleInput.sendKeys('Software Developer');
-    var typePromise = typeCheckbox.click();
+    const organizationInput = driver.findElement(seleniumWebdriver.By.name('organization.name'));
+    const titleInput = driver.findElement(seleniumWebdriver.By.name('organization.title'));
+    const typeCheckbox = driver.findElement(seleniumWebdriver.By.xpath('//input[@type="checkbox" and @value="3"]'));
+    const nextStepButton = driver.findElement(seleniumWebdriver.By.xpath(module.exports.constants.nextStepXpath));
+    const organizationPromise = organizationInput.sendKeys('MIT Media Lab');
+    const titlePromise = titleInput.sendKeys('Software Developer');
+    const typePromise = typeCheckbox.click();
     return Promise.all([organizationPromise, titlePromise, typePromise]) // eslint-disable-line no-undef
-        .then(function () {
-            nextStepButton.click().then(function () {
+        .then(() => {
+            nextStepButton.click().then(() => {
                 driver.wait(seleniumWebdriver.until
                     .elementLocated(seleniumWebdriver.By.className('address-step')));
             });
         });
```

Comment:
Does this style update also enable access to async/await? It could tidy functions like these up nicely:

```js
module.exports.fillOrganizationSlide = async function (/* ... */) {
    const organizationInput = /* ... */;
    const titleInput = /* ... */;
    const typeCheckbox = /* ... */;
    const nextStepButton = /* ... */;

    await Promise.all([
        organizationInput.sendKeys('MIT Media Lab'),
        titleInput.sendKeys('Software Developer'),
        typeCheckbox.click(),
    ]);

    await nextStepButton.click();

    driver.wait(seleniumWebdriver.until
        .elementLocated(seleniumWebdriver.By.className('address-step')));
};
```

If not, chained `.then()`s are a lot easier to read than nested ones:

```js
return Promise.all([organizationPromise, titlePromise, typePromise]) // eslint-disable-line no-undef
    .then(() => nextStepButton.click())
    .then(() => {
        driver.wait(seleniumWebdriver.until
            .elementLocated(seleniumWebdriver.By.className('address-step')));
    });
```